core/src/main/scala/cats/syntax/all.scala

```diff
@@ -94,4 +94,4 @@ trait AllSyntaxBinCompat4
 
 trait AllSyntaxBinCompat5 extends ParallelBitraverseSyntax
 
-trait AllSyntaxBinCompat6 extends ParallelUnorderedTraverseSyntax
+trait AllSyntaxBinCompat6 extends ParallelUnorderedTraverseSyntax with ParallelTraverseFilterSyntax
```

Comment:
Forgive me for being out for a while, but just curious why we still extend a binCompat trait instead of the `AllSyntax` itself?

Reply:
Ah, good catch. I agree that this should go directly on `AllSyntax`, as in #3392.

Reply:
ah of course, thanks!